gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/FlowTriggerHandler.java

```diff
@@ -94,10 +95,12 @@ public void handleTriggerEvent(Properties jobProps, DagActionStore.DagAction flo
       throws IOException {
     if (multiActiveLeaseArbiter.isPresent()) {
       MultiActiveLeaseArbiter.LeaseAttemptStatus leaseAttemptStatus = multiActiveLeaseArbiter.get().tryAcquireLease(flowAction, eventTimeMillis);
-      // TODO: add a log event or metric for each of these cases
       if (leaseAttemptStatus instanceof MultiActiveLeaseArbiter.LeaseObtainedStatus) {
         MultiActiveLeaseArbiter.LeaseObtainedStatus leaseObtainedStatus = (MultiActiveLeaseArbiter.LeaseObtainedStatus) leaseAttemptStatus;
+        this.metricContext.contextAwareCounter(ServiceMetricNames.FLOW_TRIGGER_HANDLER_LEASE_OBTAINED_COUNT);
```

Comment:
Overall flow of creating and using these metrics should be 

1. create counter at class level
2. register the metric with metric context
3. call counter.count/mark() every time you want it to increase 

Here you are creating a new one every time you mean to actually increment it. For ex: https://github.com/apache/gobblin/blob/4a9dd53b885f0166dccac5026e4e2c632c98a419/gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java#L113

Reply:
I had a different idea in mind since I only wanted to check if the `FlowTriggerHandler` gets different types of statuses or not. But I guess initializing at class level and `inc()` the counter will help gauge if the traffic distribution is happening uniformly between the hosts or not!

Reply:
Discussed offline and agreed the counter will provide more information about load balancing and relative amount of occurrence of each case